Server/Server.Test/ServerChannelMessagesTests.cpp

```diff
@@ -0,0 +1,96 @@
+#include <catch2/catch.hpp>
+#include "TestUtility.hpp"
+
+using namespace TestUtility;
+
+constexpr bool acceptingConnection = true;
```

Comment:
You are using this variable everywhere, in my opinion it will be better to move this one to TestUtility